rclcpp/src/rclcpp/context.cpp

```diff
@@ -317,7 +317,9 @@ Context::shutdown(const std::string & reason)
       rcl_ret_t rcl_ret = rcl_logging_fini();
       if (RCL_RET_OK != rcl_ret) {
         RCUTILS_SAFE_FWRITE_TO_STDERR(
+          // cppcheck-suppress unknownMacro
```

Comment:
LGTM. but quick question, why does it intend to print file and line particularly here? can we just print the error message such as `failed to finalize rcl logging` or something like that?